### PR TITLE
Email: Add server side and remove client side validation

### PR DESCRIPTION
**Pull request by hllavina**
Integrated validate email function and updated for process php script.
Removed client email validation

### CODE REVIEW

File: scripts/php/process.php [hunk 1]
```diff
@@ -119,7 +119,24 @@ private function buildMessageText () {
...
           $value = $this->cleanse($value);
-          $this->from = filter_var($value,FILTER_SANITIZE_EMAIL);
+          if($this->validateEmail($value)){
+             $this->from = filter_var($value,FILTER_SANITIZE_EMAIL);
+          } else {
+            if($_POST['recipient']=='libraries-tellus@mit.edu'){
+              // Form came from Tell Us, which allows anonymous submissions
+              $this->log->write('[Form] Email - Sender address verification failed, using stand-in address for Tell Us');
+              $this->log->write("Anonymous submission, using stand-in email address");
+              $strFrom = "tellus-lib@mit.edu";
```
Comment (matt-bernhardt) [inline scripts/php/process.php]: This should be `$this->from = ...` rather than `$strFrom = ...`

File: scripts/php/process.php [hunk 2]
```diff
@@ -119,7 +119,24 @@ private function buildMessageText () {
...
+            } else {
+              // Flag as invalid
+              $this->log->write('[Form] Email - Sender address verification failed, rejecting submission');
+              $this->log->write("[Form] Email validation failed");
+              // $valid = FALSE;
```
Comment (matt-bernhardt) [inline scripts/php/process.php]: Seeing this reminds me that I'd like us to be able to get to a better stand-in email address than my own (which is set up on L25) - @frrrances do you have a recommended address we could use?

File: scripts/php/process.php (hunk 1 above)
Comment (hllavina) [inline scripts/php/process.php]: thanks for the catch! i tested other functions viewing the debug log

**Comment by frrrances** [comment]
@matt-bernhardt I talked with Darcy and if the volume is low (once or twice a month), we can use web-lib as the email address. If the volume is higher than that, we may need to set up a list though it would be preferable to avoid that.

**Comment by hllavina** [comment]
Hi , issue #1 addressed, please review, thanks!

**Pull request merged**